templates/bundles/SyliusShopBundle/Homepage/index.html.twig

```diff
@@ -0,0 +1,72 @@
+{% extends '@SyliusShop/layout.html.twig' %}
+
+{% block content %}
+    <div class="homepage">
+        <div class="ui hidden divider"></div>
+            {{ bitbag_cms_render_media('homepage_banner') }}
```

Comment:
Should be moved to https://github.com/Sylius/Sylius/pull/11014/files#diff-cd30ca237d9d140b0ba8594e7645c29eR90